packages/salesforcedx-vscode-core/src/commands/forceCreateManifest.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { ComponentSet } from '@salesforce/source-deploy-retrieve';
+import * as fs from 'fs';
+import { join } from 'path';
+import { format } from 'util';
+import * as vscode from 'vscode';
+import { nls } from '../messages';
+import { getRootWorkspacePath } from '../util';
+
+export async function forceCreateManifest(
+  sourceUri: vscode.Uri,
+  uris: vscode.Uri[] | undefined
+) {
+  if (!uris || uris.length < 1) {
+    uris = [];
+    uris.push(sourceUri);
```

Comment:
dumb q: what use case is this supporting?

Reply:
The event that it does become available as a command palette option - @jeffb-sfdc ran into this previously and recommended this just in case.